tests/mocks/sqlite_test_db.py

```diff
@@ -17,3 +23,43 @@ def remove(cls):
     def create(cls):
         cls.remove()
         flask_migrate.upgrade()
+
+        assessment_1 = Assessment(
+            id="123e4567-e89b-12d3-a456-426655440000",
+            compliance_status="great",
+            application_id="amazing",
+        )
+        db.session.add(assessment_1)
+        db.session.commit()
+
+        sub_criteria_1 = SubCriteria(
+            sub_criteria_id="123e4567-e89b-12d3-a456-426655440001",
+            round_id="hello",
+            criteria_id="hi",
+            sub_criteria_title="something",
+        )
+        db.session.add(sub_criteria_1)
+        db.session.commit()
+
+        sub_criteria_2 = SubCriteria(
+            sub_criteria_id="123e4567-e89b-12d3-a456-426655440002",
+            round_id="ciao",
+            criteria_id="cya",
+            sub_criteria_title="nothing",
+        )
+        db.session.add(sub_criteria_2)
+        db.session.commit()
+
+        score_justification = ScoresJustifications(
+            scores_justifications_id=uuid.UUID(
+                "123e4567-e89b-12d3-a456-426655440003"
+            ),
+            created_at=dateutil.parser.parse("2022-07-07T09:11:38.240578Z"),
```

Comment:
There is no need for this library, as it appears we're just using it to parse a string into a date. We can do this with the core library e.g.
`datetime.fromisoformat('2022-07-07')`

Reply:
added strptime from datetime module to resolve this 👍🏾